docs/reference/search/explain.asciidoc

```diff
@@ -135,8 +127,7 @@ This will yield the same result as the previous request.
     Controls on which shard the explain is executed.
 
 `source`::
-    Allows the data of the request to be put in the query
-    string of the url.
+    Allows the data of the request to be put in the query string of the url.
 
```

Comment:
I'd test this out to see what it actually does. It's not clear to me what "the data of the request" is. Is it actually referring to the response?